contracts/test/AllowanceTarget/ExecuteCall.t.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.7.6;
+
+import "contracts-test/AllowanceTarget/Setup.t.sol";
+import "contracts-test/utils/BalanceSnapshot.sol";
+import "contracts-test/mocks/MockERC20.sol";
+
+contract TestAllowanceTargetExecuteCall is TestAllowanceTarget {
+    // include Snapshot struct from BalanceSnapshot
+    using BalanceSnapshot for BalanceSnapshot.Snapshot;
+
+    function testCannotExecuteByNotSpender() public {
```

Comment:
Same for test name like this, it would better to include the whole function name in there: `testCannotExecuteCallByNotSpender`

Reply:
No problem! I've filled in the full test function name.